addon/tailwind/components/bourbon-modal.css

```diff
@@ -90,86 +90,86 @@
   padding: 0;
 }
 
-.bourbon-modal__close-button:focus {
+.BourbonModal-closeButton:focus {
   outline: none;
 }
 
-.bourbon-modal__close-button svg {
+.BourbonModal-closeButton svg {
```

Comment:
This rule should probably move down one slot, so that the `.BourbonModal-closeButton` focus/hover state rules are together.